pkgs/top-level/all-packages.nix

```
@@ -20325,7 +20338,7 @@ with pkgs;
   krb5 = callPackage ../development/libraries/kerberos/krb5.nix {
     inherit (buildPackages.darwin) bootstrap_cmds;
   };
-  libkrb5 = krb5.override { type = "lib"; };
+  libkrb5 = krb5.override { type = "lib"; withVerto = false; };
```

```suggestion
  libkrb5 = krb5.override { type = "lib"; };
```
thats the default, so we don't need to repeat it here

---

Fixed.